pytest_fixtures/component/provision_libvirt.py

```diff
@@ -12,3 +12,10 @@ def module_cr_libvirt(module_target_sat, module_org, module_location):
 @pytest.fixture(scope='module')
 def module_libvirt_image(module_target_sat, module_cr_libvirt):
     return module_target_sat.api.Image(compute_resource=module_cr_libvirt).create()
+
+
+@pytest.fixture(scope='module')
+def module_libvirt_provisioning_sat(module_provisioning_sat):
+    # Configure Libvirt CR for provisioning
+    module_provisioning_sat.sat.configure_libvirt_cr()
```

Comment:
the naming of `module_provisioning_sat.sat` feels really incorrect to me, as it is not the satellite object, it returns the e.g.`module_libvirt_provisioning_setup` has satellite, domains ...